lib/src/gainmapmath.cpp

```diff
@@ -123,12 +123,13 @@ Color srgbInvOetfLUT(Color e_gamma) {
   return {{{srgbInvOetfLUT(e_gamma.r), srgbInvOetfLUT(e_gamma.g), srgbInvOetfLUT(e_gamma.b)}}};
 }
 
+// From Khronos DataFormat Spec 1.3, Section 13.3.2 - sRGB Inverse EOTF
```

Comment:
The number is correct. But I suggest removing this comment because we have a comment before `srgbInvOetf`. I want to make sure the transfer and inverse transfer come from the same source. This comment may cause confusion.

Reply:
Done.